hack/e2e-suite/update.sh

```diff
@@ -124,10 +122,15 @@ function teardown() {
   echo "Cleaning up test artifacts"
   ${KUBECTL} stop rc update-demo-kitten || true
   ${KUBECTL} stop rc update-demo-nautilus || true
+  kill -TERM "${kubectl_proxy_pid:-}" &> /dev/null || true
 }
 
 trap "teardown" EXIT
 
+# Launch a local proxy to the apiserver
+${KUBECTL} proxy --port=8011 &
```

Comment:
Why 8011?  If we have to change the default, we need to update the README.md to match.